src/styles/index.scss

```diff
@@ -1,3 +1,78 @@
 body {
   margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  &__container {
+    width: 80vmin;
+    height: 80vmin;
+    border: dotted 1vmin #000;
+    border-radius: 50%;
+    position: relative;
+  }
+
+  &__minute,
+  &__seconds,
+  &__center {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    transform-origin: bottom center;
```

Comment:
There is a mistake in the `transform-origin` property. It should be applied to the elements that are being animated (`.stopwatch__minute` and `.stopwatch__seconds`), not to the shared styles block. Move the `transform-origin` property to the blocks of `.stopwatch__minute` and `.stopwatch__seconds`.